### ISSUE TITLE
Automatic workflow for handling PAT exhaust issues

### ISSUE THREAD

**Issue by Zo-Bro-23** [anuraghazra/github-readme-stats#2427]
### Is your feature request related to a problem? Please describe.

There seem to be a lot of duplicate issues regarding the PAT rate limit.

### Describe the solution you'd like

There are a few solutions:
- [ ] A GH Actions workflow for commenting the solution to issues with "maximum retires", "PAT", "tiny.one", etc, and closing these issues.
- [x] Instead of saying "create an issue" for the SVG error message, say something like, "see the solution for this here" with a link to an issue with the solution (self deploy or other deployed URLs).
- [x] Just redirect the tiny.one link to the solution.

**Comment by rickstaa** [comment]
@Zo-Bro-23 Those are Very good suggestions. I was going insane by having to close or answer the same issue every day. 😅 We already have a way to monitor Public Vercel downtime (see https://github.com/anuraghazra/github-readme-stats/issues/1471 and https://github.com/anuraghazra/github-readme-stats/pull/2178). I also proposed an action some time ago that would update #1471 if there was downtime https://github.com/anuraghazra/github-readme-stats/issues/1989.

<img alt="Custom badge" src="https://img.shields.io/endpoint?url=https%3A%2F%2Fgithub-readme-stats-git-monitoring-github-readme-stats-team.vercel.app%2Fapi%2Fstatus%2Fup%3Ftype%3Dshields">

**Comment by airwakz** [comment]
@rickstaa  i would like to handle this issue kindle asign me as this task

**Comment by rickstaa** [comment]
@airwakz It's Amazing that you want to contribute to our project! Unfortunately, as we improved the error message in #3273, I think this feature request is no longer needed 😅. @Zo-Bro-23 I believe we can close this?

I just checked if we have other bugs or features that could qualify as Hacktoberfest entries but it looks like most have already been completed or are already assigned 🤔. We didn't have many items this year since we are focused on restructuring the code so that we can release the npm package and GitHub action. These tasks however need to be performed by the core team.
